### DISCUSSION TITLE
feat: Vanilla JS / Svelte custom components

### DISCUSSION THREAD

**Discussion by MentalGear** [payloadcms/payload#1430] [Feature Requests & Ideas]
Instead of only custom react components, it'd  be great if we could also use just vanilla JS for components, which would broaden the tools we can use (like Svelte, which is just a JS compiler), and allow more Devs to use this feature and provide custom community components.

**Comment by Snailedlt** [comment]
+1
Adding support for custom components in more frameworks, and most importantly in vanilla JS + HTML + CSS would make Payload CMS so much better!
